docs/get-started/getting-started-on-prem.md

```diff
@@ -54,14 +54,14 @@ CUDA and will not run on other versions. The jars use a maven classifier to keep
 - CUDA 11.0 => classifier cuda11-0
```

Comment:
We use classifier cuda11 for CUDA 11.0.
Should we update here or do we need to change the classifier be cuda11-0?

Reply:
@jlowe @revans2 @tgravescs any thoughts on what the cuda11 classifier should be?

Reply:
I don't think it matters too much as long as we're consistent.  Therefore I'd do whatever we did for CUDA 10.0.

Reply:
Updated to cuda11